lib/attribrutex/changeset.ex

```diff
@@ -1,4 +1,6 @@
 defmodule Attribrutex.Changeset do
+  @moduledoc false
```

Comment:
We really should add documentation to the modules that conform the package. When it is published, the docs will be available in [hexdocs.pm](https://hexdocs.pm/).